editor.planx.uk/src/@planx/components/PlanningConstraints/List.tsx

```diff
@@ -123,100 +132,71 @@ interface ConstraintListItemProps {
+          classes={{ content: classes.content }}
+          expandIcon={<Caret />}
         >
-          <Button
-            disableRipple
-            onClick={() =>
-              setShowConstraintData((showConstraintData) => !showConstraintData)
-            }
-            style={{
-              display: "flex",
-              flexDirection: "row",
-              justifyContent: "space-between",
-              width: "100%",
-              boxShadow: "none",
-              color: "#0B0C0C",
-              fontWeight: "400",
-              padding: 15,
-              paddingLeft: 20,
-            }}
-          >
-            <Box>{children}</Box>
-            <Caret
-              expanded={showConstraintData}
-              color="primary"
-              titleAccess={
-                showConstraintData ? "Less Information" : "More Information"
-              }
-            />
-          </Button>
-        </Box>
-        <Collapse in={showConstraintData}>
-          <Box py={1.5} px={2}>
-            <>
-              <Typography variant="h3" component="h4" gutterBottom>
-                {`This property ${props?.content}`}
-              </Typography>
-              {Boolean(props.data?.length) && (
-                <List
-                  dense
-                  disablePadding
-                  sx={{ listStyleType: "disc", pl: 4, pt: 1 }}
-                >
-                  {props.data &&
-                    props.data.map(
-                      (record: any) =>
-                        record.name && (
-                          <ListItem
-                            key={record.entity}
-                            dense
-                            disableGutters
-                            sx={{ display: "list-item" }}
-                          >
-                            <Typography variant="body2">
-                              {record.name}{" "}
-                              {record.name && record["documentation-url"] && (
-                                <span>
-                                  (
-                                  <Link
-                                    href={record["documentation-url"]}
-                                    target="_blank"
-                                  >
-                                    source
-                                  </Link>
-                                  )
-                                </span>
-                              )}
-                            </Typography>
-                          </ListItem>
-                        ),
-                    )}
-                </List>
-              )}
-            </>
-            <Typography variant="body2">
-              <ReactMarkdownOrHtml
-                source={props.metadata?.text?.replaceAll(
-                  "(/",
-                  "(https://www.planning.data.gov.uk/",
-                )}
-                openLinksOnNewTab
-              />
+          {children}
```

Comment:
To ensure correct responsive font size and prevent text butting up to caret icon (currently an issue in live):
`<Typography variant="body2" pr={1.5}>{children}</Typography>`